akka-actor-typed/src/main/scala/akka/actor/typed/Props.scala

```diff
@@ -179,3 +179,35 @@ object DispatcherSelector {
    */
   def sameAsParent(): DispatcherSelector = DispatcherSameAsParent.empty
 }
+
+/**
+ * Not for user extension.
+ */
+@DoNotInherit
+abstract class MailboxSelector extends Props
+
+object MailboxSelector {
+
+  /**
+   * Scala API: The default mailbox is unbounded and backed by a [[java.util.concurrent.ConcurrentLinkedQueue]]
+   */
+  def default(): MailboxSelector = DefaultMailboxSelector.empty
```

Comment:
Does it need parentheses for the Scala version?

Reply:
Hmm, maybe not, on the fence, on one hand it is a constant/no side effect on the other hand maybe that fact is an implementation detail. This is aligned with the dispatcher selector though.